rocky/rocky/views/ooi_mute.py

```diff
@@ -42,6 +42,11 @@ def post(self, request, *args, **kwargs):
         unmute = request.POST.get("unmute", None)
         selected_findings = request.POST.getlist("finding", None)
         reason = request.POST.get("reason", None)
+        end_valid_time = request.POST.get("end_valid_time", None)
+        if end_valid_time:
+            end_valid_time = datetime.strptime(end_valid_time, "%Y-%m-%dT%H:%M").astimezone(timezone.utc)
```

Comment:
The expiration is given in UTC. `astimezone` assumes that the naive datetime is in the system timezone, not in UTC. To add UTC timezone to a naive datetime you need to use replace as described in https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone:

```suggestion
            end_valid_time = datetime.strptime(end_valid_time, "%Y-%m-%dT%H:%M").replace(tzinfo=timezone.utc)
```